### ISSUE TITLE
Deleted scans are running in background

### ISSUE THREAD

**Issue by Hacker22o2** [yogeshojha/rengine#157]
Hi Team,

Suppose we gave 1st scan for yahoo.com and gave a 2nd scan for example.com, and we deleted the yahoo.com scan, in the background yahoo.com scan is still running. In the screenshot below yahoo.com scan is deleted but its still running

![image](https://user-images.githubusercontent.com/33136022/88474789-9654f800-cf47-11ea-879d-298535c26bd2.png)

**Comment by S3ntago** [comment]
How can you check the running process in the background as shown in the screenshot for reNgine ?

**Comment by JoshuaMart** [comment]
Hello,
I think this is normal and not expected behavior, simply because removing the stain in the UI does not stop the script.

This may be implemented when Rengine uses Celery or something similar.

Regards

**Comment by yogeshojha** [comment]
You are absolutely right @JoshuaMart. I am working on migrating to celery, that is the first priority as of now. Scheduling scans in the future and canceling scans is what are the must features as requested from multiple people.

Hopefully, by this weekend, I will be able to complete this.

**Comment by Hacker22o2** [comment]
Yes @JoshuaMart, i too thought that only, lets wait for celery update.

**Comment by yogeshojha** [comment]
Thanks for your patience. 😄 

This is finally released. Please check version 0.4 🥳 A lot of new features have been added as well. Give it a try
